pkgs/by-name/cb/cbqn/package.nix

```
@@ -75,7 +80,7 @@ stdenv.mkDerivation {
   preBuild =
     ''
       mkdir -p build/singeliLocal/
-      cp -r ${sources.singeli.src}/* build/singeliLocal/
+      cp -r build/singeliSubmodule/* build/singeliLocal/
```

If you remove the `mkdir` before, it can become

```suggestion
      mv build/singeliSubmodule/ build/singeliLocal
```

Should also work for the other instances of this!